src/experimental/information_estimates.cpp

```diff
@@ -112,29 +108,27 @@ std::tuple<int, kt_double> InformationEstimates::calculateMutualInformation(std:
                 }
 
                 // Mutual information of cell x, y given a set of measurements                
-                updateCellMutualInformation(1.0 - cell_mutual_inf, cell);
+                updateCellMutualInformation(1.0 - cell_mutual_inf, cell); 
             }
         }
-        kt_double map_mut_info = calculateMapMutualInformation(); 
+        // Mutual information provided by this laser scan
+        kt_double laser_mut_info = calculateLaserMutualInformation();
+        scans_mut_info.emplace_back(laser_mut_info);
```

Comment:
@kmilo7204 IIUC this is computing `I(M, Z[n])`. I'm not so sure that `arg min I(M, Z[n])` is equivalent to 
`arg min I(M, Z) - I(M, Z \ {Z[n]})`. Measurement outcome histograms and probabilities of occupancy aren't the same if the measurement set changes, AFAIK. Mind to explain?